network-grpc/src/client.rs

```diff
@@ -1,6 +1,6 @@
 use crate::gen::{self, node::client as gen_client};
 
-use chain_core::property::{self, Deserialize, FromStr, HasHeader, Serialize};
+use chain_core::property::{self, Block, Deserialize, FromStr, HasHeader, Serialize};
```

Comment:
Rebase this as well to eliminate cross-pollination from other branches. I'll continue reviewing from the clean branch.